app/assets/javascripts/admin/templates/user-index.hbs

```diff
@@ -221,7 +221,13 @@
 
   <div class='display-row'>
     <div class='field'>{{i18n 'admin.user.admin'}}</div>
-    <div class='value'>{{model.admin}}</div>
+    <div class='value'>
+      {{#if model.admin}}
```

Comment:
I have extracted a helper `i18n-yes-no` in https://github.com/discourse/discourse/pull/4629/commits/e17bd6174c3497ccb349d1a697fb013156f10213. The prefix `i18n` announces that translation is taking place and `yes-no` (as suggested by @ZogStriP) is less technical than `boolean`.